.circleci/config.yml

```diff
@@ -62,7 +52,6 @@ jobs:
       - run: *test
       - run: *spec_test
       - run: go test -v -race ./release/verval
-      - setup_remote_docker
```

Comment:
You should be able to remove this for the other jobs as well (golang 1.14 etc)

Reply:
✅